tests/app/celery/test_letters_pdf_tasks.py

```diff
@@ -491,7 +491,7 @@ def test_process_letter_task_check_virus_scan_passed(
     source_bucket_name = current_app.config["LETTERS_SCAN_BUCKET_NAME"]
     target_bucket_name = current_app.config[bucket_config_name]
 
-    conn = boto3.resource("s3", region_name="ca-central-1")
+    conn = boto3.resource("s3")
```

Comment:
Why is the region no longer needed?  Did the api for boto3 change?

Reply:
From [the docs](https://boto3.amazonaws.com/v1/documentation/api/1.17.58/reference/core/session.html#boto3.session.Session.resource) it looks like it looks the region name is picked up first from: `botocore.client.Config` then `Env Variables`. If the region name is passed into `boto3.resource()` it will take highest precedence. I'm guessing since this is letter code it was just leftovers. L585 still has `eu-west-1`.